tests/unit/utils/sparqlhandler/test_csvw_inspector.py

```diff
@@ -55,49 +54,52 @@ def test_get_primary_catalog_metadata_key_error():
 def test_detect_csvw_type_qb_dataset():
     """
     Tests the detection of the csvw_type of an input metadata json file that has a CsvwRdfManager
-    object created from it, in this case to correctly detect a QbDataset csvw type from its csvw_state.
+    object created from it, in this case to correctly detect a QbDataset csvw type from its csvw_inspector.
     """
     csvw_metadata_json_path = (
         _test_case_base_dir
         / "pivoted-single-measure-dataset"
         / "qb-id-10004.csv-metadata.json"
     )
-    csvw_rdf_manager = get_csvw_rdf_manager(csvw_metadata_json_path)
+    csvw_rdf_manager = CsvwRdfManager(csvw_metadata_json_path)
 
-    csvw_type = csvw_rdf_manager.csvw_state.csvw_type
+    csvw_type = csvw_rdf_manager.csvw_inspector.csvw_type
     assert csvw_type == CSVWType.QbDataSet
 
 
-def test_csvw_type_key_error():
```

Comment:
Where has this key error test gone? Do we lose anything without it?

Reply:
Reinstated